Docs: Fix running luxon from dev console

gricey432 opened the pull request:
Fix #1174 by ensuring this path is absolute and resolves to https://moment.github.io/luxon/global/luxon.js
Currently it tries to request https://moment.github.io/global/luxon.js which doesn't exist

gricey432 commented:
I don't think I did a good job here, doesn't seem to have fixed the associated ticket

h-h-h-h commented:
> Currently it tries to request https://moment.github.io/global/luxon.js which doesn't exist

This is still what happens. Didn't the merged change become live?

gricey432 commented:
Yeah I think I fixed this in the wrong spot. I fixed it on the homepage https://moment.github.io/luxon/#/
The same change will need to be made to the api-docs pages
